### PR TITLE
feat(content): Remove Gegno Quarg Landables from Quarg Missions

### PR DESCRIPTION
**Pull request by Saugia**
**Content (Artwork / Missions / Jobs)**

## Summary
Due to the nature of the Gegno's First Contact mission and Quarg-related systems, it is possible to get Quarg missions at times that aren't desired, such as `Ringworld Debris` or `Ask Quarg about Coalition Early`, where they could speak the normal way they would anywhere else, in Gegno systems, where they don't speak your language at the start (let alone fluently only soon after).

Although it's possible to get these missions from the Korath ringworld, it makes more sense that they could speak the player's language than it would the much farther away Gegno ring, and even moreso non-station Quarg planets. This PR simply removes these missions from offering on Quarg landables in Gegno space.

In the future, I may add separate conversations for these missions if you do land on these places in Gegno space, and that would provide some interesting lore-building that each Quarg ring has somewhat separate cultures and discussions. @Arachi-Lover 

## Save File
~~I don't have a save that deep in the game, sorry.~~

### CODE REVIEW

**Comment by Arachi-Lover** [comment]
Pasting what I said on Discord on the different Quarg responses idea.

I'd rather not have different groups have different responses, as that'd really just make it so you never get the "true" reaction (that being, some Quarg who does have the relevant Coalition-Quarg conflict held as a major thing), because the Coalition Quarg don't have places to be in of their own, with the whole stolen rings thing you know.
The Quarg who lost their rings to the Coalition would number in over a trillion, and they've since been orphaned of a ring and scattered throughout the galaxy. I don't find it any difficult to believe that a decent portion of any ring's population consists of such ring orphans.
For me it's important the message conveyed by the Quarg when asked this comes from a Quarg that was affected by the Coalition, meaning one such ring orphan.
I don't mind the mission being gated from offering in X or Y place before you've done relevant stuff there, like with the Gegno Quarg, but I don't want the Gegno Quarg, or any other Quarg, speaking for the Coalition Quarg in this.

**Comment by Saugia** [comment]
Hm, okay. In such a case, would you prefer that either I keep the missions from offering on these worlds, or a branch in that coalition mission where they basically say the same things except taking into account the language discrepancy?

**Comment by Arachi-Lover** [comment]
To be clear, I was talking specifically about the `Ask Quarg about Coalition Early` mission in that comment, I don't mind different regions' Quarg reacting to the ringworld debris within their own manner as the language barrier permits.

For the Coalition mission, I think it'd be really difficult to give essentially the same speech but within the language restrictions, I don't even think the player could really ask the Quarg that can't speak the human language about the Coalition and get them to understand the question. Maybe just preventing it from offering there, for the moment at least.

File: data/kahet/kahet missions.txt [hunk 1]
```diff
@@ -247,6 +247,8 @@ mission "Ringworld Debris: Quarg"
 	invisible
 	source
 		attributes "quarg"
+		not
+			attributes "gegno quarg"
```
Comment (Anarchist2) [inline data/kahet/kahet missions.txt:251]:
```suggestion
		not attributes "gegno quarg"
```

File: data/quarg/quarg missions.txt [hunk 2]
```diff
@@ -236,6 +236,8 @@ mission "Ask Quarg About Coalition Early"
 	source
 		attributes "quarg"
 		attributes "station"
+		not
+			attributes "gegno quarg"
```
Comment (Anarchist2) [inline data/quarg/quarg missions.txt:240]:
```suggestion
		not attributes "gegno quarg"
```

File: data/quarg/quarg missions.txt [hunk 3]
```diff
@@ -282,6 +284,8 @@ mission "Quarg Pug Arfecta Warning"
 	source
 		attributes "quarg"
 		attributes "ringworld"
+		not
+			attributes "gegno quarg"
```
Comment (Anarchist2) [inline data/quarg/quarg missions.txt:288]:
```suggestion
		not attributes "gegno quarg"
```

**Comment by Anarchist2** [comment]
No changes to the main Quarg first contact mission?

**Comment by Saugia** [comment]
>No changes to the main Quarg first contact mission?

Last time I brought up this, Derpy had said that it isn't a problem with the Gegno Quarg so much as it is a problem with how the first Quarg mission works. I will add it now regardless because even if it's possible the player visits the Gegno Quarg before any other Quarg, they probably shouldn't have a casual conversation as is done with first contact.

**Review by W1zrad — APPROVED** [review]
Looks good to me. @Anarchist2 speak within the next week or so or forever hold your peace

**Pull request merged**